src/foremast/utils/awslambda.py

```diff
@@ -118,3 +120,36 @@ def add_lambda_permissions(function='',
 
     LOG.debug('Related StatementId (SID): %s', statement_id)
     LOG.info(response_action)
+
+def remove_all_lambda_permissions(app_name='', env='', region='us-east-1'):
+    """removes all foremast-* permissions from lambda
+
+    Args:
+        app_name (str): Application name
+        env (str): AWS environment
+        region (str): AWS region
+    """
+    session = boto3.Session(profile_name=env, region_name=region)
+    lambda_client = session.client('lambda')
+
+    lambda_arn = get_lambda_arn(app_name, env, region)
+    lambda_alias_arn = get_lambda_alias_arn(app_name, env, region)
+    arns = (lambda_arn, lambda_alias_arn)
+
+    for arn in arns:
+        try:
+            response = lambda_client.get_policy(FunctionName=arn)
+        except boto3.exceptions.botocore.exceptions.ClientError as error:
+            LOG.info("No policy exists for function %s, skipping deletion", arn)
+            LOG.debug(error)
+            return
+
+        policy_json = json.loads(response['Policy'])
+        LOG.debug("Found Policy: %s", response)
+        for perm in policy_json['Statement']:
+            if perm['Sid'].startswith('foremast-') or app_name+'_' in perm['Sid']:
```

Comment:
I would like to see:
```python
legacy_sid_check = app_name + '_' in perm['Sid']
```
This would make the if statement read a little easier. Also, should it be `in`, or `.startswith()`?
```python
FOREMAST_PREFIX = 'foremast-'

legacy_prefix = app_name + '_'
if perm['Sid'].startswith(FOREMAST_PREFIX) or perm['Sid'].startswith(legacy_prefix):
```